### ISSUE TITLE
Relationships can be navigated by relying on a subset of SQL 92？

### ISSUE THREAD

**Issue by ZTY-star** [ADLINK-IST/opensplice#135]
Where is SQL 92 defined? Is it in the IDL file?

**Comment by vivekpandey02** [comment]
No, SQL is not defined in IDL file. In Vortex Opensplice DDS there is a DbmsConnect Service by using that you can map you DDS topics to SQL table. 

For details please see "chapter 10: The DbmsConnect Service" in "OpenSplice_DeploymentGuide.pdf" document
